pkg/skaffold/server/server.go

```diff
@@ -190,7 +203,19 @@ func newGRPCServer(preferredPort int, usedPorts *util.PortSet) (func() error, in
 }
 
 func newHTTPServer(preferredPort, proxyPort int, usedPorts *util.PortSet) (func() error, error) {
-	mux := runtime.NewServeMux(runtime.WithProtoErrorHandler(errorHandler), runtime.WithMarshalerOption(runtime.MIMEWildcard, &runtime.JSONPb{OrigName: true, EmitDefaults: true}))
+	mux := runtime.NewServeMux(
```

Comment:
Do we still need the `runtime.WithProtoErrorHandler(errorHandler)` to be part of our ServeMux?  It seems this snippet is from - https://grpc-ecosystem.github.io/grpc-gateway/docs/development/grpc-gateway_v2_migration_guide/#we-now-use-the-camelcase-json-names-by-default, just want to make sure we dont' lose any options/behavior that was present in the past marshalling

Reply:
Fixed. ErrorHandler removed. It is no more required in the version 2. We are still using the similar marshalling options to OrigName, EmitDefaults, ie. UseProtoNames and EmitUnpopulated